multichain-testing/scripts/fund-provision-pool.ts

```diff
@@ -56,6 +79,7 @@ async function fundProvisionPool(args: {
       throw new Error(`Transaction failed: ${resultData['raw_log']}`);
     }
 
+    await waitForBlock(1);
```

Comment:
We only reach here if the last transaction succeeds. That -bblock mode and this one is too, so there should be no need to wait.

Please remove so that if it does fail we can correct the theory.

Reply:
Agree. I was mainly after a race between tx inclusion and the bank balance update. You're right that `-bblock` should ensure this doesn't happen.

If we start to see **Error: No balances found**, we can update this check to use the more robust `waitUntilAccountFunded`.